fastlane/Fastfile

```diff
@@ -205,8 +214,8 @@ platform :ios do
             body: trigger_task_body,
             headers: authorization_header
         )
-        
-        puts triggered_task
+        # {"data":{"trigger":{"task":{"id":"6498826324803584"}}}}
+        puts triggered_task["data"]["trigger"]["task"]["id"]
```

Comment:
I think actually it might be helpful to generate the url here rather than just printing id